src/fsharp/TastOps.fs

```diff
@@ -2927,7 +2927,7 @@ let TryBindTyconRefAttribute g (m:range) (AttribInfo (atref, _) as args) (tcref:
         | None -> None
 #endif
     | ILTypeMetadata (TILObjectReprData(_, _, tdef)) -> 
-        match TryDecodeILAttribute g atref tdef.CustomAttrs with 
+        match TryDecodeILAttribute g atref (tdef.GetFilteredCustomAttributes(g.ilg)) with 
```

Comment:
I think this call can just be `tdef.CustomAttrs`, no need to filter here  as we only filter when searching for `ObsoleteAttribute`